File: Ioana_Hisham/src/main/java/ql/ast/expressions/Binary/GreaterThanOrEqual.java
package ql.ast.expressions.binary;

import ql.ast.expressions.Expression;
import ql.visitors.ExpressionVisitor;

public class GreaterThanOrEqual extends Binary {

    public GreaterThanOrEqual(int lineNumber, Expression leftHandSide, Expression rightHandSide) {
        super(lineNumber, leftHandSide, rightHandSide);
    }

    @Override
    public <T> T accept(ExpressionVisitor<T> expressionVisitor) {
        return expressionVisitor.visit(this);
    }
}

File: Erik_Vincent/Assignment1/Assignment1/Model/QL/AST/Expression/GreaterThanOrEqual.cs
namespace Assignment1.Model.QL.AST.Expression
{
    public class GreaterThanOrEqual : Binary
    {
        public GreaterThanOrEqual(int lineNumber, IExpression left, IExpression right) : base(left, right)
        {
            _lineNumber = lineNumber;
        }

        public override void Accept(IExpressionVisitor visitor) => visitor.Visit(this);
    }
}